src/js/actions/export.js

```diff
@@ -416,40 +454,60 @@ define(function (require, exports) {
 
     /**
      * After start up, ensure that generator is enabled, and then initialize the export service
+     *
+     * If in debug mode, do a pre check in case we're using a remote connection generator
+     * 
      * @return {Promise}
      */
     var afterStartup = function () {
-        return descriptor.playObject(generatorLib.getGeneratorStatus())
-            .bind(this)
-            .then(function (status) {
-                var enabled = objUtil.getPath(status, "generatorStatus.generatorStatus") === 1;
-                if (!enabled) {
-                    log.info("Enabling Generator...");
-                    return descriptor.playObject(generatorLib.setGeneratorStatus(true))
-                        .catch(function (e) {
-                            throw new Error("Could not enable generator", e);
-                        });
-                }
-            })
-            .delay(500)
-            .then(function () {
-                _exportService = new ExportService();
+        // helper function to wait for the service init, and set the service "available" in our store
+        var _initService = function () {
+            return _exportService.init()
+                .bind(this)
+                .then(function () {
+                    log.info("Export: Plugin connection established");
+                    return _setServiceAvailable.call(this, true);
+                });
+        };
 
-                return _exportService.init()
+        // In debug mode we should first do an abbreviated test
+        // to see if the plugin is already running (perhaps on a remote generator connection)
+        var preCheck;
+        if (globalUtil.debug) {
+            log.debug("Doing a quick check to see if Export plugin is available");
```

Comment:
Ditto. I think we should only add log messages like this if we think they will be useful for some engineers. It doesn't seem like this passes that test.